theseus/src/launcher/java.rs

```diff
@@ -1,17 +0,0 @@
-use crate::launcher::LauncherError;
-use std::process::Command;
-
-
-pub fn check_java() -> Result<String, LauncherError> {
-    let child = Command::new("java")
-        .arg("-version")
-        .output()
-        .map_err(|inner| LauncherError::ProcessError {
-            inner,
-            process: "java".into(),
-        })?;
-
-    let output = String::from_utf8_lossy(&child.stderr);
-    let output = output.trim_matches('\"');
-    Ok(output.into())
-}
```

Comment:
The reason this module existed is so we could detect outside installs of Java/query the version. Are you sure we want to delete this?

Reply:
From what I could tell, this was already dead code and only checked for the global Java version. I could see if it were public or used in the first place, but it's current existence is a bit difficult to justify (and it probably would be rewritten later anyway if someone doesn't know it exists and needs it's functionality). Usually, it's a good idea to implement private utilities as needed. It's your call, though.

Reply:
Yeah, I guess you can remove it. If we need it, we can just go back in commit history and get it.